reactive/calico.py

```diff
@@ -89,6 +90,7 @@ def install_calico_service(etcd):
         'etcd_ca_path': ETCD_CA_PATH,
         'etcd_cert_path': ETCD_CERT_PATH,
         # specify IP so calico doesn't grab a silly one from, say, lxdbr0
+        # REVIEW(Matt): I like the use of a fixed IP, but I'm worried that the IP might change (e.g. reboot and re-DHCP).  Not sure how much of a concern that is in a Juju environment.
```

Comment:
They can change on reboots. Juju handles this by updating unit data. We need to make sure we re-render on changes